packages/web/src/components/collection/CollectionCard.test.tsx

```diff
@@ -89,12 +93,63 @@ describe('CollectionCard', () => {
     ).toBeInTheDocument()
   })
 
-  it('shows the number of reposts and favorites with the correct screen-reader text', () => {
-    renderCollectionCard()
-    expect(screen.getByTitle('Reposts')).toBeInTheDocument()
-    expect(screen.getByText('10')).toBeInTheDocument()
+  it('hidden collections are rendered correctly', () => {
+    renderCollectionCard({
+      reduxState: {
+        collections: { entries: { 1: { metadata: { is_private: true } } } }
+      }
```

Comment:
I like this pattern. Very clean per test

Reply:
thanks! i wonder also if we can add helpers for quickly adding users, tracks collections, so you dont actually need to know the full redux shape.